kitsune/wiki/forms.py

```diff
@@ -158,20 +153,34 @@ def clean_slug(self):
         return slug
 
     def clean(self):
-        c = super(DocumentForm, self).clean()
-        locale = c.get("locale")
+        cdata = super(DocumentForm, self).clean()
+        locale = cdata.get("locale")
 
         # Products are required for en-US
-        products = c.get("products")
-        if locale == settings.WIKI_DEFAULT_LANGUAGE and (not products or len(products) < 1):
+        product_ids = cdata.get("products")
+        products = Product.objects.filter(id__in=[int(product_id) for product_id in product_ids])
+        if locale == settings.WIKI_DEFAULT_LANGUAGE and (not product_ids or len(product_ids) < 1):
             raise forms.ValidationError(PRODUCT_REQUIRED)
 
         # Topics are required for en-US
-        topics = c.get("topics")
-        if locale == settings.WIKI_DEFAULT_LANGUAGE and (not topics or len(topics) < 1):
+        topic_ids = cdata.get("topics")
+        topics = Topic.objects.filter(id__in=[int(topic_id) for topic_id in topic_ids])
```

Comment:
Same here for `topic_ids`. It needs to take into account that `topic_ids` can be `None`, and I think we should use `Topic.active.filter(...)` here as well?